content/influxdb/v2.0/reference/flux/flux-vs-influxql.md

```diff
@@ -0,0 +1,408 @@
+  )
+  |> map(fn: (r) => ({
+    _time: r._time,
+    _value: (r._value_mem / r._value_proc) / 1000000
+  })
+)
+```
+
+### Sort by tags
+InfluxQL's sorting capabilities are very limited, allowing you only to control the
+sort order of `time` using the `ORDER BY time` clause.
+The Flux [`sort()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/sort)
+sorts records based on list of columns.
+Depending on the column type, records are sorted lexicographically, numerically, or chronologically.
+
+```js
+from(bucket:"example-bucket")
+  |> range(start: -12h)
+  |> filter(fn: (r) =>
+    r._measurement == "system" and
+    r._field == "uptime"
+  )
+  |> sort(columns:["region", "host", "_value"])
+```
+
+### Group by any column
+InfluxQL lets you group by tags or by time intervals, but nothing else.
+Flux lets you group by any column in the dataset, including `_value`.
+Use the Flux [`group()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/group/)
+to define which columns to group data by.
+
+```js
+from(bucket:"example-bucket")
+  |> range(start: -12h)
+  |> filter(fn: (r) => r._measurement == "system" and r._field == "uptime" )
+  |> group(columns:["host", "_value"])
+```
+
+### Window by calendar months and years
+InfluxQL does not support windowing data by calendar months and years due to their varied lengths.
+Flux supports calendar month and year duration units (`1mo`, `1y`) and lets you
+window and aggregate data by calendar month and year.
+
+```js
+from(bucket:"example-bucket")
+  |> range(start:-1y)
+  |> filter(fn: (r) => r._measurement == "mem" and r._field == "used_percent" )
+  |> aggregateWindow(every: 1mo, fn: mean)
+```
+
+### Work with multiple data sources
+InfluxQL can only query data stored in InfluxDB.
+Flux can query data from other data sources such as CSV, PostgreSQL, MySQL, Google BigTable, and more.
+Join that data with data in InfluxDB to enrich query results.
+
+- [Flux CSV package](/influxdb/v2.0/reference/flux/stdlib/csv/)
+- [Flux SQL package](/influxdb/v2.0/reference/flux/stdlib/sql/)
+- [Flux BigTable package](/influxdb/v2.0/reference/flux/stdlib/experimental/bigtable/)
+
+<!-- -->
+```js
+import "csv"
+import "sql"
+
+csvData = csv.from(csv: rawCSV)
+sqlData = sql.from(
+  driverName: "postgres",
+  dataSourceName: "postgresql://user:password@localhost",
+  query:"SELECT * FROM example_table"
+)
+data = from(bucket: "example-bucket")
+  |> range(start: -24h)
+  |> filter(fn: (r) => r._measurement == "sensor")
+
+auxData = join(tables: {csv: csvData, sql: sqlData}, on: ["sensor_id"])
+enrichedData = join(tables: {data: data, aux: auxData}, on: ["sensor_id"])
+
+enrichedData
+  |> yield(name: "enriched_data")
+```
+
+_For an in-depth walkthrough of querying SQL data, see [Query SQL data sources](/influxdb/v2.0/query-data/flux/sql/)._
+
+### DatePart-like queries
+InfluxQL doesn't support DatePart-like queries that only return results during specified hours of the day.
```

Comment:
"DatePart" is a specific category of SQL functions that anyone coming from a SQL context will understand. This is also a keyword that people use when searching for these types of functions.

Reply:
Gotcha. Thought about folks coming from InfluxQL or Flux...in case someone's just learning Flux and happened upon "Possible with Flux"

bc we refer to "hourSelection" function and hour is a specified "DatePart" maybe we rephrase to "Query by hour" (and then refer to "DatePart" in the description in case folks search for that)?